pynotify/models.py

```diff
@@ -55,9 +55,11 @@ class AdminNotificationTemplate(BaseTemplate):
     Attributes:
         slug: Template slug, with which this template can be referred to.
         is_active: Flag that switches on/off creating notifications from this template.
+        send_push: Flag that switches on/off sending push notifications from this template.
     """
     slug = models.SlugField(max_length=200, unique=True, verbose_name=_l('slug'))
     is_active = models.BooleanField(default=True, verbose_name=_l('is active'))
+    send_push = models.BooleanField(default=False, verbose_name=_l('send push notification'))
```

Comment:
Please generate and commit PO and MO file:
```
make po
make mo
```

Reply:
Also normally, I would require you to update `HISTORY.md` file, but since we'll be releasing soon, I can do that for you during release. Also, add yourself to `AUTHORS.md`, if you want.